images/jenkins/config/template.apko.yaml

```diff
@@ -1,35 +1,45 @@
 contents:
   packages:
     # jenkins and openjdk provided by extra_packages
+    - jenkins-docker
 
 accounts:
   groups:
     - groupname: jenkins
-      gid: 65532
+      gid: 1000
   users:
     - username: jenkins
-      uid: 65532
-  run-as: 65532
-
-work-dir: /app
+      gid: 1000
+      uid: 1000
+      homedir: "/var/jenkins_home"
```

Comment:
CI and me both seem to be getting this message here? Should I be upgrading something?

```
│ yaml: unmarshal errors:
│   line 8: field homedir not found in type types.User
```

Reply:
working on rolling out all the necessary changes, this had to be added in apko https://github.com/chainguard-dev/apko/pull/1146